plugins/woocommerce/tests/legacy/unit-tests/util/install.php

```diff
@@ -166,4 +166,12 @@ function ( $tables ) {
 
 		$this->assertContains( 'some_table_name', WC_Install::get_tables() );
 	}
+
+	/**
+	 * Test that dbDelta is a noop on an installed site.
+	 */
+	public function test_dbDelta_is_a_noop() {
```

Comment:
Thinking of [this guidance](https://github.com/woocommerce/woocommerce/blob/7.5.0/plugins/woocommerce/tests/README.md?plain=1#L130):

> No new tests should be added here, ever; existing test classes shouldn't get new tests either.

Would it be better to move this test to `WC_Install_Test`?

Reply:
True, true. Moved the test in [abe3e28](https://github.com/woocommerce/woocommerce/pull/37277/commits/abe3e28bf1aa73b67a06f9b776aac2b809277ce8).